### DISCUSSION TITLE
`a->b` as edge

### DISCUSSION THREAD

**Discussion by hchiba1** [pg-format/pg-formatter#29] [Ideas]
By minor modifications of `Edge` and `UNQUOTED`, `a->b` can be interpreted as an edge.

**Comment by nichtich** [comment]
This seems useful but then the character sequences `--` and `->` (and `<-`) cannot be used in unquoted identifiers. I'd prefer to keep allowed characters very broad to support at least plain IRIs. `>` could be excluded but `--` is used (actually we have such URIs in our database). Better requires space around direction.

**Comment by nichtich** [comment]
Both `a<-b` and `a ->b` may be allowed still, as `<` and `>` are not allowed in plain URIs anyway. As

`a->b`

read as

- node id `a-`
- `>`
- `b`

is invalid grammar, we may also allow this, if peggy grammar can handle such case.

**Comment by hchiba1** [comment]
Followed by a general discussion:
* #37
